godtools/Views/Cells/HomeToolTableViewCell.swift

```diff
@@ -76,7 +76,7 @@ class HomeToolTableViewCell: UITableViewCell {
         titleLabel.isEnabled = isAvailableInPrimaryLanguage
         titleLabel.text = resource.localizedName(language: primaryLanguage)
         
-        languageLabel.text = resource.isAvailableInLanguage(parallelLanguage) ? parallelLanguage!.localizedName() : nil
+        languageLabel.text = resource.isAvailableInLanguage(parallelLanguage) ? parallelLanguage!.localizedName : nil
```

Comment:
Sometimes I think that we can these kind of scenarios, where maybe it would be nice to also decorate, so, instead of making this check here have a function on the class or in the resource, what do you think? (this is something that applies in a lot of places)